Sources/ReleaseServer/ReleaseServer.WebApi/Models/ValidationResultModel.cs

```diff
@@ -0,0 +1,9 @@
+namespace ReleaseServer.WebApi.Models
+{
+    public class ValidationResultModel
```

Comment:
Same as above: If there is no other class `ValidationResult`, why do you need a "Model" of it?

Reply:
I removed the "Model" suffix, see https://github.com/Traeger-GmbH/release-server/commit/014b059c85cec990a6c6c065dd406057b121f649